### DISCUSSION TITLE
Sending Limit Static or Variable

### DISCUSSION THREAD

**Discussion by CoffeeShogun** [postalserver/postal#2800] [Help with using Postal] [answered]
Hello!

We have been using Postal lightly and are planning to ramp up usage over the next 6 months to include many more customer domains. I did not configure the server initially but maintain it now so please forgive my ignorance. 

Currently, the Send Limit tab for our production server reports that an unlimited number of e-mails can be sent every 60 minutes. This is ideal as we don't have a solid number yet on how many emails to expect per hour. Is this number something that could change in the future? If so, what kind of criteria would force that change so we can be cognizant of that? We want to make sure that the amount of emails that could be sent won't be a bottleneck in the future. 

Thanks for any assistance!

**Comment by willpower232** [comment, marked as answer]
Its a setting only used to manually limit how many messages can be sent. There is no automation at all and it will stay at unlimited unless someone changes the setting.

![image](https://github.com/postalserver/postal/assets/1619102/84af503e-867f-4ae9-b715-2c109314626e)

**Reply by CoffeeShogun** [reply]
Awesome to hear, thank you so much sorry for the simple question 😅
